src/lib/variants.ts

```diff
@@ -2,6 +2,7 @@ import { EpicTargeting, ViewLog, WeeklyArticleHistory } from '../components/Cont
 import { shouldThrottle, shouldNotRenderEpic } from '../lib/targeting';
 import { getArticleViewCountForWeeks } from '../lib/history';
 import { getCountryName, countryCodeToCountryGroupId } from '../lib/geolocation';
+import { isRecentOneOffContributor } from './targeting';
```

Comment:
Does this function belong in here now it's no longer part of targeting?

Reply:
I've moved `isRecentOneOffContributor` and `daysSince` to a new lib file `dates.ts` as they're used by targeting and variants.